### ISSUE TITLE
Add "opens in a new tab" symbol to data sources in header

### ISSUE THREAD

**Issue by bothness** [ONSdigital/explore-local-statistics-app#330]
In general, the data sources in the header should be improved, especially when there are multiple sources for an indicator. In all cases, we should have the external link symbol for all links that open in a new tab.

Also fix this for the fact that multi-source datasets source formatting is silly at the moment, eg: https://www.ons.gov.uk/explore-local-statistics/indicators/gross-median-weekly-pay
